docs/src/assets/scss/syntax-highlighter.scss

```diff
@@ -120,3 +120,12 @@ pre[class*="language-"] {
         width: 1.2em;
     }
 }
+
+.incorrect,
+.correct {
+    @media all and (min-width: 768px) {
+        pre.line-numbers-mode {
+            padding-bottom: 4rem;
+        }
+    }
+}
```

Comment:
It looks good if we give the bottom padding for the code block as `4rem` and the playground button bottom as `1rem`

<img width="1106" alt="image" src="https://github.com/eslint/eslint/assets/30730124/690ccc57-1c99-43c5-9371-cd7965009970">

Reply:
Done! 
![WhatsApp Image 2023-07-24 at 10 57 04 PM](https://github.com/eslint/eslint/assets/86398394/38124067-5834-4fa4-b2cb-cc2b81f6a718)

Reply:
have kept the bottom position 1.5rem in PC size because it is touching the scrollbar.

![Screenshot 2023-07-24 231146](https://github.com/eslint/eslint/assets/86398394/602fe26c-70c6-405a-83e5-09fd132facb5)